wpp-generator-v2.php

```diff
@@ -1,15 +1,6 @@
 <?php
 
 /**
- * WPPG Next
- * 
- * @package           MXSFWNWPPGNext ???
- * @author            Maksym Marko
- * @copyright         2024 Free Software Foundation
- * @license           GPL-2.0-or-later
- * 
- * @wordpress-plugin
- * 
  * Plugin Name:       WPPG Next
```

Comment:
```suggestion
 * Plugin Name:       WPPG Next
```
This is a suggestion block I'm talking about.

Reply:
Yes, good!